pyproject.toml

```diff
@@ -349,11 +349,47 @@ template = "development/towncrier_template.j2"
 start_string = "<!-- towncrier release notes start -->"
 issue_format = "[#{issue}](https://github.com/nautobot/nautobot/issues/{issue})"
 
-[tool.towncrier.fragment.added]
-[tool.towncrier.fragment.changed]
-[tool.towncrier.fragment.dependencies]
-[tool.towncrier.fragment.deprecated]
-[tool.towncrier.fragment.fixed]
-[tool.towncrier.fragment.housekeeping]
-[tool.towncrier.fragment.removed]
-[tool.towncrier.fragment.security]
+[[tool.towncrier.type]]
```

Comment:
Did you test this syntax? I've had some trouble with towncrier's documentation not aligning with their released code.

Reply:
Testing it to generate pre-2.0.4 release notes appears to work. Thanks for checking!